src/databricks/labs/ucx/queries/assessment/main/40_4_remaining_tables.sql

```diff
@@ -0,0 +1,4 @@
+/* --title 'List of remaining tables in HMS' --width 6 */
```

Comment:
Could you make this part of the migration dashboard?

Reply:
Moved